### DISCUSSION TITLE
Should we have `set_pixel()` ?

### DISCUSSION THREAD

**Discussion by villares** [py5coding/py5generator#197] [Ideas]
I know now we have advanced NumPy arrays of pixels and all, but the `get()`  function looks a bit lonely now.... 

http://py5coding.org/reference/sketch_get.html?highlight=get

I  know it is highly inefficient comparing to access to `pixels[]` and `np_pixels[]` but maybe it should be the "entry level" way of doing old `set()` as `set_pixel()`. 

Maybe `get()` should be called `get_pixel()`?

**Comment by hx2A** [comment]
py5 has no `set()` method because it would conflict with the Python builtin but there is a `get()` method...I agree, it would have been a better decision to rename `set()` to `set_pixel()` and then also rename `get()` to the analogous `get_pixel()`. That's a better API design than providing a `get()` without a `set()`. This is an easy change to make. Will open an issue for this now.
